webpack.config.prd.js

```diff
@@ -16,7 +16,7 @@ module.exports = env => {
     }),
     new webpack.BannerPlugin({ 
       banner: header,
-      raw: true, 
+      raw: false, 
```

Comment:
This needs to remain `true` in order to keep the Tampermonkey headers in the release file.

Reply:
if I put it to true it has actually no headers, this is because the update of the obfuscator. When i try it with false it adds the headers correctly and the rest of the script is obfuscated.

Reply:
@Mardaneus86  do you experience something different when you compile it with `npm run build:production` ?

Reply:
Hm, will check tomorrow, out of time for today! Thanks for your efforts.

Reply:
The upgrade to webpack v4 seems to break the way the BannerPlugin is working. Reverting to v3 resolves the issue, so this can be investigated at a later point since it doesn't change functionality. The header comments are necessary for the production build.

I'll add an issue to the backlog for it.

Reply:
ok, i have downgraded, but once i use 3.8.1 the code it is not getting obfuscated

Reply:
Obfuscation is not necessary, since this enables traceability between the code in the repo and the distributed file, only minification is enough. I think the dependency on `webpack-obfuscator` was added in an early version and never got removed again. Webpack v4 uses the `UglifyJsPlugin` by default in production mode, which isn't what we want for this.